lib/player.rb

```diff
@@ -1,29 +1,32 @@
-require 'pry'
 require "./lib/scrabble.rb"
+require "./lib/tile_bag"
 
 class Player
-  attr_accessor :name, :player_words
+  attr_accessor :name, :player_words, :tile_tray
 
+#initializes a new player
```

Comment:
I love all the comments you've added for every method, this is a really good practice!